packages/agents/sequencer/src/lib/operations/auctions.ts

```diff
@@ -39,6 +41,15 @@ export const storeBid = async (bid: Bid, _requestContext: RequestContext): Promi
     });
   }
 
+  // check if bid router version is compatible with hosted sequencer
+  const checkVersion = compareVersions(bid.routerVersion, config.supportedBidVersion!);
```

Comment:
So we will error on any version mismatch? Do we want to use `~` to support patch versions? https://www.npmjs.com/package/compare-versions

Reply:
updated in human-readable form